src/faultlog/faultlog_main.cpp

```diff
@@ -140,6 +176,9 @@ int main(int argc, char** argv)
         CLI::App app{"Faultlog tool"};
         app.set_help_flag("-h, --help", "Faultlog tool options");
 
+        initPHAL();
```

Comment:
Please use a try, catch block to address the execptions

Reply:
already in try catch block